src/components/internals/calendar-day/calendar-day.mod.css

```diff
@@ -0,0 +1,53 @@
+.wrapper {
+  list-style-type: none;
+  display: inline-block;
+  margin: 2px 1%;
+  width: 12%;
+}
+
+.base {
+  text-align: center;
+  padding: 8px 0;
+  cursor: default;
+}
+
+.day {
+  composes: base;
+  border-radius: 4px;
+}
+
+.heading {
+  composes: base;
+  color: #999;
+}
+
+.spacing {
+  composes: base;
+}
+
+/* States */
+
+.highlighted {
+  background-color: #e6e6e6;
```

Comment:
Just curious towards the long term vision of these colors? Are they gonna be a token?

Reply:
Ah thanks. These should use tokens already. I skipped the tokens during implementation with the intention to add them before opening the PR... which I forgot about in excitement :P

Reply:
I replaced the plain values with tokens now. Thanks again for the reminder, I had already done it in other files but must have skipped this one.